tests/lib/rules/require-default-props.js

```diff
@@ -1718,6 +1728,17 @@ ruleTester.run('require-default-props', rule, {
           column: 3
         }
       ]
+    },
+    {
+      code: [
+        'import type { ImportedType } from "fake";',
+        'type Props = ImportedType;',
+        'function Hello(props: Props) {',
+        '  return <div>Hello {props.name.firstname}</div>;',
+        '}'
+      ].join('\n'),
+      parser: 'babel-eslint',
+      errors: []
```

Comment:
When you say "update this", are you referring to the rule, to throw an error if it can't find a variable? If so, I think that should be a separate PR.

If you are referring to the test case when you say "update this", then when making the requested changes, the invalid test still passes with `errors: []` so I am not sure there is much difference between what you are suggesting and what I added.